sass/components/list.sass

```diff
@@ -0,0 +1,22 @@
+$list-border: 1px solid $border !default
+$list-background-color: $white !default
+$list-radius: $radius !default
+
+$list-item-border: 1px solid $border !default
+$list-item-hover-color: $background !default
+
+.list
+  background-color: $list-background-color
+  border: $list-border
+  border-radius: $list-radius
+  .list-item
+    color: black
+    cursor: default
+    display: flex
```

Comment:
Remove this.

Reply:
without `display: flex`
![image](https://user-images.githubusercontent.com/30751516/37329682-fcb27018-269e-11e8-89ab-c7e52c725bbf.png)